### DISCUSSION TITLE
shading on hover - for tabs / panels etc.

### DISCUSSION THREAD

**Discussion by benkoshy** [retypeapp/retype#344] [Ideas]
It would be very nice if there was some type of signifier upon hovering - i.e. a slight color change etc. Am I missing something? Is there any way this can be done?

I looked through the retype repo and I could not find where this change could be made or suggested:

![image](https://user-images.githubusercontent.com/15097447/175434134-3bac5013-0d47-4386-80ae-141405f7bd62.png)

Without some shading or some other signifier, users might not realise that this is even a tab.

any advice would be beneficial.

chrs

**Comment by geoffreymcgill** [comment]
The blue underline switches to gray and the gray text changes to blue when the Tab is hovered. This is a somewhat minimal Tab user-experience, although relatively common design, for example in Material Design themes used in Android and other OS's.

https://user-images.githubusercontent.com/62210/175436508-b5e29696-c998-4841-b909-e851d80d7385.mov

**Reply by benkoshy** [reply]
Agreed. I would argue that the signifier for that affordance isn't strong enough - notwithstanding the minimalist designs and the patterns of  other frameworks. even the very lightest and slightest background shading on hover would more than suffice - if you still want to retain the minimalist approach.

There may be consequential complications which I am not privvy to with that approach. 

I offer them as feedback, directly from a user - here's a work in progress docs site we're working on for the pagy open source project:

 https://benkoshy.github.io/pagy/docs/api/frontend/

**Comment by benkoshy** [comment]
@geoffreymcgill 

Same theme, but slightly different context. I feel the [signifiers](https://www.interaction-design.org/literature/topics/signifiers) are a little too weak. The panels are not clearly distinguishable. At least on my browser (firefox),  OS, my screen and operating system, and the lighting conditions in my environment. Changing screens makes a marginal difference.

A possible solution worth investigating: change background color on hover - a very light shading to serve the minimalist approach.

See here for a [live demo]( https://benkoshy.github.io/pagy/docs/extras/searchkick/#methods)

![image](https://user-images.githubusercontent.com/15097447/184254921-4999ede6-1018-448d-a647-0f354d04de0c.png)


That's my two cents anyways.
